### DISCUSSION TITLE
github怎么改名字？

### DISCUSSION THREAD

**Discussion by AcheronBlackswanForever** [share121/inter-knot#2948] [其他]
![v2-940bb5f2f89ca67ccbb589345a06a686_r](https://github.com/user-attachments/assets/25e885b0-52e9-4e13-9f57-0940fdc45b7f)

好久不用这个网站，为了绳网登上来了，发现自己名字很怪异？？

**Comment by qk0809** [comment]
希腊奶               （头像怎么换？

**Reply by AcheronBlackswanForever** [reply]
头像点一下头像就换...（有点像废话）

**Comment by AcheronBlackswanForever** [comment]
改成功了，竟然藏在account里面
